test/health-spec.js

```diff
@@ -0,0 +1,12 @@
+var assert = require('assert'),
+  health = require('../src/health');
+
+describe('health', function() {
```

Comment:
Sometimes `function()` is used, sometimes the ES6 syntax is used. Is there a specific reason for that ?